lib/game_engine.rb

```diff
@@ -17,13 +17,13 @@ def play
     loop do
```

Comment:
Maybe turn loop into a while loop and have a condition instead of a break statement.